tests/test_subpixmaxima.py

```diff
@@ -10,145 +12,113 @@
 _TORCH_DEVICE = "cuda" if torch.cuda.is_available() else "cpu"
 
 
-def test_subpixmaxima():
-    data_transform = []
-    data_transform.append(
+def test_generate_keypoints():
+    data_transform = [
         iaa.Resize({"height": 384, "width": 384})
-    )  # dlc dimensions need to be repeatably divisable by 2
+    ]  # dlc dimensions need to be repeatably divisable by 2
     imgaug_transform = iaa.Sequential(data_transform)
     dataset = HeatmapDataset(
         root_directory="toy_datasets/toymouseRunningData",
         csv_path="CollectedData_.csv",
         header_rows=[1, 2],
         imgaug_transform=imgaug_transform,
     )
-    SubPixMax = SubPixelMaxima(
-        output_shape=(96, 96),
-        output_sigma=torch.tensor(1.25, device=_TORCH_DEVICE),
-        upsample_factor=torch.tensor(2, device=_TORCH_DEVICE),
-        coordinate_scale=torch.tensor(4, device=_TORCH_DEVICE),  # 2 ** 2
-        confidence_scale=torch.tensor(1, device=_TORCH_DEVICE),
-        threshold=None,
-        device=_TORCH_DEVICE,
-    )
     example = dataset.__getitem__(idx=0)
-    maxima, confidence = SubPixMax.run(
-        example["heatmaps"].unsqueeze(0).to(_TORCH_DEVICE)
+    assert example["keypoints"].shape == (torch.Size([34]))
+    heatmap_gt = example["heatmaps"].unsqueeze(0)
+    keypts_gt = example["keypoints"].unsqueeze(0).reshape(1, 17, 2)
+    assert heatmap_gt.shape == (1, 17, 96, 96)
+    heatmap_torch = generate_heatmaps(
```

Comment:
do we mind replacing it with `kornia`'s `render_gaussian_2d`? it'll lighten our repo and be a more tested version of `generate_heatmaps`. Another plus is that the heatmap is generated and with the same machinery that then does the subpixmax ops.
https://kornia.readthedocs.io/en/v0.2.0/_modules/kornia/geometry/dsnt.html#render_gaussian_2d

Reply:
yep that's a good idea